pkg/datapath/linux/datapath_test.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2018 Authors of Cilium
+// Copyright 2019 Authors of Cilium
```

Comment:
Change to 2018-2019

Reply:
This is new code, diff just abuses a random old file